apps/src/netsim/NetSimVizWire.js

```diff
@@ -38,15 +38,6 @@ var NetSimVizWire = (module.exports = function(localNode, remoteNode) {
    */
   this.line_ = jQuerySvgElement('path').appendTo(root);
 
-  /**
-   * @type {jQuery} wrapped around a SVGTextElement
-   * @private
-   */
-  this.questionMark_ = jQuerySvgElement('text')
-    .text('?')
-    .addClass('question-mark')
-    .appendTo(root);
```

Comment:
Do we want to leave this around so that A/B mode continues to work as-is if you don't have auto-receive turned on in the level config?